crypto/src/main/java/org/web3j/crypto/Blob.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.crypto;
+
+import org.apache.tuweni.bytes.Bytes;
+
+public class Blob {
```

Comment:
This class should have hashCode and equals implemented so Blob instances can be compared based on data, ideally also some helpful toString